source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -131,6 +131,12 @@ export abstract class CardElement {
     private _defaultRenderedElementDisplayMode: string = null;
     private _padding: PaddingDefinition = null;
 
+    public elementId: string;
```

Comment:
This needs to be removed. CardElement already has an `id` property.

Reply:
P0: Id property already exists - remove

Reply:
In progress.